example/pubspec.lock

```diff
@@ -14,7 +14,7 @@ packages:
       name: characters
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.0.0"
+    version: "1.1.0-nullsafety.3"
```

Comment:
Why did you update this?

Reply:
I run `flutter pub get` & this updated these packages. Do you want to remove these updates from this PR?

Reply:
We can update if it's just for a local fork. We shouldn't update if you're planning to raise a PR to the original repo.